server/models/article.js

```diff
@@ -38,6 +57,42 @@ articleSchema.statics.urisAreEqual = (uri1, uri2) => {
   return normalizeURI(uri1) === normalizeURI(uri2);
 };
 
+articleSchema.methods.getMercuryInfo = function getMercuryInfo() {
+  const encodedURI = encodeURIComponent(this.uri);
+  return fetch(`https://mercury.postlight.com/parser?url=${encodedURI}`,
+  { headers: { 'Content-Type': 'application/json', 'x-api-key': process.env.MERCURY_API_KEY } })
+  .then((res) => {
+    return res.json();
+  }).then((json) => {
+    if ( // Mercury error conditions:
```

Comment:
Weird... I guess not really anything to do about it if we can't get more info than that.